src/web/app/pages-instructor/instructor-session-edit-page/instructor-session-edit-page.component.html

```diff
@@ -20,6 +20,7 @@
                              (deleteCurrentQuestionEvent)="deleteExistingQuestionHandler(i)"
                              [(formModel)]="questionEditFormModels[i]"></tm-question-edit-form>
     </div>
+    <a class="scrollup-link" (click)="scrollToTopOfPage()" id="scrollUp">Back to top of page</a>
```

Comment:
A minor change request. Can we change this to the following?
```suggestion
    <a class="scrollup-link" (click)="scrollToTopOfPage()">
      Back to Top <i class="fas fa-arrow-up"></i>
    </a>
```

Looks something like this:
<img width="188" alt="Screen Shot 2022-05-16 at 2 03 58 PM" src="https://user-images.githubusercontent.com/60355570/168529138-a11d04f6-061c-42b2-817c-f806abd6b18d.png">

Rationale: There is already a back to top link styled this way in the instructor help page. Better user experience if we stay consistent across pages. I also think this looks slightly better too.